osf/utils/migrations.py

```diff
@@ -205,7 +205,13 @@ def create_schema_block(state, schema_id, block_type, display_text='', required=
         help_text=unescape_entities(help_text),
         registration_response_key=registration_response_key,
         schema_block_group_key=schema_block_group_key,
-        example_text=unescape_entities(example_text)
+        example_text=unescape_entities(
```

Comment:
seems like we should unescape < and > for `display_text` and `help_text`, too?

Reply:
concur